core/system/src/mpu/vmpu_armv7m_debug.c

```diff
@@ -30,19 +30,21 @@ void debug_mpu_config(void)
     /* CTRL */
     ctrl = MPU->CTRL;
     dprintf("\n\r");
-    dprintf("  Background region %s\n\r", ctrl & MPU_CTRL_PRIVDEFENA_Msk ?
-                                          "enabled" : "disabled");
-    dprintf("  MPU %s @NMI, @HardFault\n\r", ctrl & MPU_CTRL_HFNMIENA_Msk ?
-                                             "enabled" : "bypassed");
-    dprintf("  MPU %s\n\r", ctrl & MPU_CTRL_PRIVDEFENA_Msk ?
-                                  "enabled" : "disabled");
+    dprintf("  Background region %s\n\r",
+            ctrl & MPU_CTRL_PRIVDEFENA_Msk ?
+                "enabled" : "disabled");
```

Comment:
I would put the elements of a ternary operator always on the same line (here and in following occurrences)